test/rulesets/Base/base.jl

```diff
@@ -121,6 +121,28 @@
         )
     end
 
+if VERSION ≥ v"1.4"
+    @testset "evalpoly" begin
```

Comment:
If we do the following we can follow the same indenting everywhere
```suggestion
    VERSION ≥ v"1.4" && @testset "evalpoly" begin
```